tests/base/pom.xml

```diff
@@ -131,4 +136,16 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>mixed-cluster</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <kc.test.server>cluster</kc.test.server>
+            </properties>
+        </profile>
+    </profiles>
+
```

Comment:
Guessing this is a left-over from when the tests where in `tests/base` and can be removed now
```suggestion
```

Reply:
This was not intentional. Pedro added it during POC and I missed this was there